<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paginadehipertrofia</class>
 <widget class="QMainWindow" name="paginadehipertrofia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>901</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>901</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>462</width>
    <height>901</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1, 55, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>800</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 55, 51);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="perfil">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>810</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botaoperfil{
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>fotos/perfil.png</normaloff>fotos/perfil.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="config">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>810</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botaoconfig{
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>fotos/config.png</normaloff>fotos/config.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>81</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="treino">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>810</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botaotreino{
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>fotos/treino.png</normaloff>fotos/treino.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>81</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>810</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botaomenu{
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>fotos/menu.png</normaloff>fotos/menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>81</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>99</y>
      <width>461</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 65, 61);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>468</width>
       <height>2468</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>2450</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 65, 61);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 221, 166);
font: 75 19pt &quot;Georgia&quot;;</string>
         </property>
         <property name="text">
          <string>Segunda</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>441</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fotos/quadro perna segunda(hiper.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>410</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 221, 166);
font: 75 19pt &quot;Georgia&quot;;</string>
         </property>
         <property name="text">
          <string>Terça</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>470</y>
           <width>441</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fotos/quadro peito terça  (hiper.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>780</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 221, 166);
font: 75 19pt &quot;Georgia&quot;;</string>
         </property>
         <property name="text">
          <string>Quarta</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>840</y>
           <width>441</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fotos/quadro costas quarta (hiper.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>1160</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 221, 166);
font: 75 19pt &quot;Georgia&quot;;</string>
         </property>
         <property name="text">
          <string>Quinta</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>1230</y>
           <width>441</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fotos/quadro perna quinta (hiper.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>1540</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 221, 166);
font: 75 19pt &quot;Georgia&quot;;</string>
         </property>
         <property name="text">
          <string>Sexta</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>1600</y>
           <width>441</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fotos/quadro peito sexta (hiper.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>1910</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 221, 166);
font: 75 19pt &quot;Georgia&quot;;</string>
         </property>
         <property name="text">
          <string>Sábado</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>1970</y>
           <width>441</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fotos/quadro costas sabado (hiper.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>2290</y>
           <width>381</width>
           <height>141</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fotos/info (hiper.png</pixmap>
         </property>
        </widget>
        <widget class="QPushButton" name="botaohiperA">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>340</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#botaohiperA{
background-color:#0A5B56;
color:#FFDDA6;
font: 75 10pt &quot;Georgia&quot;;
}</string>
         </property>
         <property name="text">
          <string>Exercícios - A</string>
         </property>
        </widget>
        <widget class="QPushButton" name="botaohiperB">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>720</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#botaohiperB{
background-color:#0A5B56;
color:#FFDDA6;
font: 75 10pt &quot;Georgia&quot;;
}</string>
         </property>
         <property name="text">
          <string>Exercícios - B</string>
         </property>
        </widget>
        <widget class="QPushButton" name="botaohiperC">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>1090</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#botaohiperC{
background-color:#0A5B56;
color:#FFDDA6;
font: 75 10pt &quot;Georgia&quot;;
}</string>
         </property>
         <property name="text">
          <string>Exercícios - C</string>
         </property>
        </widget>
        <widget class="QPushButton" name="botaohiperA_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>1490</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#botaohiperA_2{
background-color:#0A5B56;
color:#FFDDA6;
font: 75 10pt &quot;Georgia&quot;;
}</string>
         </property>
         <property name="text">
          <string>Exercícios - A (2)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="botaohiperB_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>1850</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#botaohiperB_2{
background-color:#0A5B56;
color:#FFDDA6;
font: 75 10pt &quot;Georgia&quot;;
}</string>
         </property>
         <property name="text">
          <string>Exercícios - B (2)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="botaohiperC_2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>2220</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#botaohiperC_2{
background-color:#0A5B56;
color:#FFDDA6;
font: 75 10pt &quot;Georgia&quot;;
}</string>
         </property>
         <property name="text">
          <string>Exercícios - C (2)</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 65, 61);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fotos/hipertrofia.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="actionhipertrofia">
   <property name="text">
    <string>hipertrofia</string>
   </property>
  </action>
  <action name="actionfor_a">
   <property name="text">
    <string>força</string>
   </property>
  </action>
  <action name="actiondefini_o">
   <property name="text">
    <string>definição</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
